fastmri_examples/adaptive_varnet/eval_pretrained_adaptive_varnet.py

```diff
@@ -0,0 +1,254 @@
+import pathlib
+from argparse import ArgumentParser
+
+import fastmri
+import matplotlib.pyplot as plt
+import numpy as np
+import pytorch_lightning as pl
+import torch
+from fastmri import evaluate
+from fastmri.data.mri_data import fetch_dir
+from fastmri.data.transforms import MiniCoilTransform
+from fastmri.pl_modules import ActiveVarNetModule, FastMriDataModule
+from subsample import create_mask_for_mask_type
+
+
+def str2bool(v):
+    if isinstance(v, bool):
+        return v
+    if v.lower() in ("yes", "true", "t", "y", "1"):
+        return True
+    elif v.lower() in ("no", "false", "f", "n", "0"):
+        return False
+    else:
+        raise ValueError("Boolean value expected.")
+
+
+def entropy(prob_mask: torch.Tensor):
+    ent = -(prob_mask * prob_mask.log() + (1 - prob_mask) * (1 - prob_mask).log())
+    ent[prob_mask == 0] = 0
+    ent[prob_mask == 1] = 0
+    return ent
+
+
+def save_images(recon):
+    for i, tensor in enumerate(recon):
+        plt.imshow(tensor.detach().cpu().numpy(), cmap="gist_gray")
+        plt.colorbar()
+        plt.savefig(f"/scratch/lep/active_acq/recon_{i}.png")
```

Comment:
Should we make this save path an arg, or alternatively save to some parent of the checkpoint dir?